.github/workflows/client-compatibility-tests.yml

```diff
@@ -38,11 +38,38 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  changes:
+    name: Find changed files
+    runs-on: ubuntu-latest
+    outputs:
+      should-run: ${{ steps.filter.outputs.every_changed == 'true' || steps.filter.outputs.any_changed == 'true' }}
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
+      - name: Filter other changes
+        uses: ./.github/actions/filter-changes
+        id: filter
+        with:
+          # Filter files that match **every** pattern in this list
+          every-filter: |
+            - '**'                     # Initially include all files
+            - '!**/*.md'               # Exclude all markdown files (lowercase)
+            - '!**/*.MD'               # Exclude all markdown files (uppercase)
+            - '!**/.github/**'         # Exclude all changes under the .github directory
+            - '!testdata/**'           # Exclude all testdata directories
+            - '!tools/**'              # Exclude all tools directories            
+          # Filter files that match **any** of these patterns
+          any-filter: |
+            - '.github/workflows/client-compatibility-tests.yml'
+            - '.github/actions/notify-slack-jobs-result/**'
```

Comment:
See my comment at https://github.com/smartcontractkit/chainlink/pull/14585/files#diff-31bd198ce2c6956ed10bdfe25a233b755f3a201ad094a67c449115ef76d9ff43R75 on why this is a bad idea.